docs/particles/functional.rst

```diff
@@ -39,18 +37,16 @@ The full symbol of the particle can be found using
 Particle properties
 ===================
 
-The `~plasmapy.particles.atomic.atomic_number` and
-`~plasmapy.particles.atomic.mass_number` functions are analogous to the
+The |atomic_number| and |mass_number| functions are analogous to the
 corresponding attributes in the |Particle| class.
```

Comment:
Even though some of these substitutions are used only once, I'm astonished at how much more readable this documentation is.  There's also the side benefit that this would be much easier to rewrite if we were to decide to.